src/experimental/react/controllers/animation-map-controller.js

```diff
@@ -163,21 +165,19 @@ export default class AnimationMapController extends PureComponent {
   }
 
   _animateViewportProp(nextProps) {
-    if (this.props.animateViewport !== true) {
-      return;
-    }
-    const startViewport = this._extractViewportFromProps(this.props);
-    const endViewport = this._extractViewportFromProps(nextProps);
-    if (this._didViewportAnimatePropChanged(startViewport, endViewport)) {
-      const animationInterval = this._createAnimationInterval();
-      this.setState({
-        animationInProgress: true,
-        animationT: 0.0,
-        animationStartViewport: startViewport,
-        animationEndViewport: endViewport,
-        animationInterval,
-        animatedViewport: startViewport
-      });
+    if (this._isViewportAnimationEnabled()) {
+      const startViewport = this._extractViewportFromProps(this.props);
+      const endViewport = this._extractViewportFromProps(nextProps);
+      if (this._didViewportAnimatePropChanged(startViewport, endViewport)) {
+        const animationInterval = this._createAnimationInterval();
+        this.setState({
+          animationT: 0.0,
```

Comment:
Let's not abbreviate identifiers, unless we have a strong reason. Why is `animationT` better than `animationTime`?